packages/loader/container-loader/src/loader.ts

```diff
@@ -418,7 +418,10 @@ export class Loader implements IHostLoader {
 					});
 				}
 			})
-			.catch((error) => {});
+			.catch((error) => {
+				// If an error occured while resolving the container request, then remove it from the cache.
+				this.containers.delete(key);
```

Comment:
I think you need to rethrow error here :)
I want to say - let's also add UTs to capture bugs like that, but given that we are removing caching (or on the path to remove it) - it's probably fine to skip it.